src/components/CustomCollection/CustomCollection.module.scss

```diff
@@ -15,3 +15,12 @@
   @include u-padding-x(2);
   @include u-margin-top(2);
 }
+
+button.collectionSettingsDropdown,
+.usa-prose > button.collectionSettingsDropdown {
```

Comment:
nit: we shouldn't need this selector (`.usa-prose > button.collectionSettingsDropdown`) unless there's a button with this class that's a direct descendant of `.usa-prose`, which there doesn't seem to be

Reply:
thanks! went ahead and removed -- can you re-approve when you have a moment?